conf/app.routes

```diff
@@ -7,8 +7,8 @@ GET         /return-after-gg-creds-created        uk.gov.hmrc.agentsubscriptionf
 
 GET         /business-type                        uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showBusinessTypeForm
 POST        /business-type                        uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.submitBusinessTypeForm
-GET         /business-details                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showBusinessDetailsForm(businessType: Option[String])
-POST        /business-details                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.submitBusinessDetailsForm(businessType: Option[String])
+GET         /business-details                     uk.gov.hmrc.agentsubscriptionfrontend.controllers.BusinessIdentificationController.showBusinessDetailsForm(businessType: uk.gov.hmrc.agentsubscriptionfrontend.models.IdentifyBusinessType)
```

Comment:
May be able to avoid fuly qualified type here because it's imported in the `routesImport` in build.sbt